pkg/hubble/relay/observer/server_test.go

```diff
@@ -339,6 +342,146 @@ func TestGetFlows(t *testing.T) {
 	}
 }
 
+func TestGetFlows_follow(t *testing.T) {
```

Comment:
IMHO the goal of this test is not immediately obvious at first glance, so suggesting a comment that should help to that end
```suggestion
// test that Relay pick up a joining Hubble peer.
func TestGetFlows_follow(t *testing.T) {
```